### DISCUSSION TITLE
Libvips-dev dependence in Debian package

### DISCUSSION THREAD

**Discussion by jpferrero** [photoprism/photoprism#4304] [General]
I notice that libvips-dev is a dependency in the latest .deb package. Is that correct? I believe that the correct dependency should be libvips42 instead of the development package of libvips.

Thanks,

**Comment by lastzero** [comment]
In theory, yes. In practice, we are always going to support the latest/most recent version (available on Debian/Ubuntu) and want these packages to be compatible with all/most Debian-based Linux distributions e.g. `libvips-dev` automatically installs `libvips42` (Ubuntu Jammy and Mantic) or `libvips42t64` (Noble) as a dependency.

Although the dev package installs a few additional header files and development dependencies such as gettext and pkgconf, these should not significantly increase the overall download/install size given the other dependencies. If you would like to contribute packages that reference the exact packages/versions for each distribution and release, you are of course welcome to do so.

**Reply by jpferrero** [reply]
Thank you for your answer. In my case, installing libvips-dev install lots of additional packages (excluding recommends). I'd rather use the tar.gz version instead of installing .deb package and manually install libvips42, which it seems mostly the same.

**Reply by lastzero** [reply]
We have updated our documentation and release notes to mention that these packages are intended for experienced users and maintainers of third-party integrations only. Especially the .deb and .rpm packages are work in progress and we only have limited time to improve them (though other maintainers and our community could help with that):
- https://dl.photoprism.app/pkg/linux/README.html
